server/autotest_server/__init__.py

```diff
@@ -257,9 +257,13 @@ def _clear_working_directory(tests_path: str, test_username: str) -> None:
     if test_username != getpass.getuser():
         sticky_cmd = f"sudo -u {test_username} -- bash -c 'chmod -Rf -t {tests_path}'"
         chmod_cmd = f"sudo -u {test_username} -- bash -c 'chmod -Rf ugo+rwX {tests_path}'"
+        clean_tmp_cmd = (
```

Comment:
This is good, just make sure to update the function docstring to note this additional action.